components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/base/includes/activities_info.html

```diff
@@ -77,10 +77,16 @@
                 new_results = -1;   // hide notification number
             }
             OME.displayStatus(new_results, inprogress);
+            if (recursive) {
+                setTimeout(function(){
+                    OME.activitiesUpdate(true);
+                }, 5000);
+            }
 
         }).error(function() {
```

Comment:
Here you need to check status of the response otherwise this display timeout even there were 500